docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Installing-Loris-in-Depth.md

```diff
@@ -0,0 +1,195 @@
+# Installing LORIS In-Depth
+
+This guide assumes an installation on an Ubuntu server. 
+
+If you are using CentOS, please visit the CentOS installation guide for 
+instructions on installation.
+
+We also have instructions for how to set up LORIS for macOS. However, only
+ Ubuntu and CentOS are officially supported by the team so we may be unable to
+ help you with issues arising on other systems.
+
+
+When you've completed this guide, further set up is required. Please visit
+[the Setup page](https://github.com/aces/Loris/wiki/Setup) in order to be
+able to use LORIS properly.
+
+## Prequisities
+
+### Server Stack
+
+LORIS requires a LAMP stack in order to run, specifically:
+* Apache 2.4
+* MySQL 5.7 (or MariaDB 10.1) (or higher)
+* PHP 7.2 (or higher)
+
+Additionally, LORIS requires the following package managers:
+* npm 8.0 (or higher)
+* composer
```

Comment:
Any minimal version?
Some people had problem with older version recently.

Reply:
what problems?

Reply:
`make` failing with error. See mailing list.

Reply:
I'm not sure what version is needed. I'll make a separate issue for this because this will need to be fixed in other places than this document.

Reply:
#5845